site/layouts/partials/promo-banner.html

```diff
@@ -1,5 +1,5 @@
 <section class="promo-banner headless">
-  👉&nbsp;<strong>NEW:</strong>&nbsp;Automatic performance & error tracing on every browser check.
+  <div>👉 <strong>NEW:</strong> Automatic performance & error tracing on every browser check.</div>
```

Comment:
To improve a11y, is a good practice to put emojis inside `span` elements and set `role="img"`

```html
<span role="img" aria-label="finger point right">👉</span>
```

Reply:
BTW, I think I would prefer the `&nbsp;` instead of normal space. It's more explicit and will avoid automatic trailing from some code editors (IMO)

Reply:
Good to know, will fix that!